Node.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BST
+{
+    class Node
+    {
+        public int key;
+        public Node left, right;
+
+        public Node(int item)
+        {
+            key = item;
+            left = right = null;

```

Comment:
Aren't these attributes set to null initially, I mean, during the construction of object? Is there any need for explicitly initializing them to null?